packages/harper-ls/package.yaml

```diff
@@ -0,0 +1,45 @@
+---
+name: harper-ls
+description: The Grammar Checker for Developers
+homepage: https://github.com/chilipepperhott/harper-ls
+licenses:
+  - Apache-2.0
+languages:
+  - Markdown,
+  - Rust,
+  - TypeScript,
+  - JavasCript,
+  - Python,
+  - Go,
+  - C,
+  - C++,
+  - Ruby,
+  - Swift,
```

Comment:
Swift is not an accepted filetype (will make the check fail, when it is triggered).

If you install the yaml lsp, you will get completion and diagnostics that validate for the mason packages (which is how I could tell)